package.json

```diff
@@ -164,6 +164,12 @@
     "opentype.js": "^1.3.3",
     "optimize-css-assets-webpack-plugin": "^5.0.4",
     "percy": "^0.28.4",
+    "postcss": "^8.2.2",
+    "postcss-advanced-variables": "^3.0.1",
+    "postcss-loader": "^4.1.0",
+    "postcss-nested": "^5.0.3",
+    "postcss-preset-env": "^6.7.0",
+    "postcss-scss": "^3.0.4",
```

Comment:
Still not sure why we need scss here. Seems like you are only using nested and variables feature, which you don't need scss for.

Reply:
we have used `@for` where needed interpolation of the `$i` variable and it is not possible without using `postcss-scss`. Build will throw errors.

I would recommend keeping this around in case we use more of interpolation.